src/Controller/Helpers/StructuredDataHelper.php

```diff
@@ -108,6 +111,33 @@ public function getSchemaForNonActorContribution(Contribution $contribution): ar
         return $this->schemaHelper->buildSchemaForContributor($contribution);
     }
 
+    public function getSchemaForCharacter(Profile $profile, CoreEntity $programme, bool $showParent = true)
+    {
+        $schema = $this->schemaHelper->buildSchemaForCharacter($profile);
+        if ($showParent) {
+            if (!$programme->isTlec()) {
+                $programme = $programme->getParent();
+            }
+            $schema['isPartOf'] = $this->getSchemaForProgrammeContainer($programme);
```

Comment:
Having a look at `$programme->getParent()` it seems like it could return null but `getSchemaForProgrammeContainer` doesn't accept null so probably need to do something like this to be safer
```suggestion
            if ($programme) {
                $schema['isPartOf'] = $this->getSchemaForProgrammeContainer($programme);
            }
```